go-controller/.golangci.yml

```diff
@@ -55,9 +55,11 @@ linters-settings:
         alias: apierrors
       - pkg: k8s.io/apimachinery/pkg/util/errors
         alias: kerrors
-      # Controller Runtime
       - pkg: sigs.k8s.io/controller-runtime
         alias: ctrl
+      # Other frequently used deps
+      - pkg: github.com/ovn-org/libovsdb/ovsdb
+        alias: ""
```

Comment:
isn't it better to do with `libovsdb` instead?
I think we are using:

- libovsdb
- libovsdbops
- libovsdbutils
- libovsdbclient
etc?
this is a nit I am fine with ovsdb as well whatever works

Reply:
No, `libovsdb` is already `github.com/ovn-org/libovsdb`.

Again, this is not about best form primarily, but homogeneity.

Reply:
An here I am just covering egregious cases where the lack of uniformity actually becomes confusing or causes duplicate imports. Can be improved over time though.